include/mega/megaapp.h

```diff
@@ -389,6 +389,11 @@ struct MEGA_API MegaApp
 
     virtual void getmiscflags_result(error) { }
 
+    virtual void backupput_result(error, handle /*backup id*/) { }
```

Comment:
Should we start using `Error` in these new callbacks, rather than `error`? Maybe a reference o a unique_ptr, if dynamic allocation is needed. The advantage would be to get the additional information of the error at upper layers when the API provides so.

Reply:
Solved it